### DISCUSSION TITLE
Dynamic QR

### DISCUSSION THREAD

**Discussion by skyway22** [lyqht/mini-qr#24] [Ideas]
Could it be done?

**Comment by WhiteHatRootkitJedi** [comment]
That's exactly what I'm using this for! I haven't finished yet, but I'm connecting this to the Shlink API which I've deployed in a docker container. Basically, you type in a url into mini-qr, and the qr code is generated with the shortlink that shlink creates. Then you can modify the shortlink later!

**Reply by DerBoki** [reply]
Just today i was planning on doing the same, but i'm using a self-hosted yourls. Unfortunately i have no idea how to make this mini-qr run on all-inkl.com webhosting. I have ssh access, but i have no idea what's needed as i am not really into the node and vue, etc side of things and there is just nothing except the docker compose. I'd like if there was a simple installation guide actually.😅

**Reply by lyqht** [reply]
With the new docker images, does it solve your issues?

**Comment by EduNetEurope** [comment]
Hello @lyqht,
your QR-generator is absolutely wonderful, thank you!
There is another [open-source QR-generator](https://github.com/giandonatoinverso/PHP-Dynamic-Qr-code), which lacks the nice styling options yours offers, but has dynamic QR codes, statistics and supports [vcards](https://github.com/lyqht/mini-qr/discussions/60), etc.
The combination of the two with an additional folder/subfolder structure would be truly amazing.
Is there any chance the codes can be combined?

**Reply by lyqht** [reply]
hello there, thank you for your feedback!

i don't think i'll combine the source codes. 
- I may add dynamic qr codes in the future, but i would prefer to implement it from scratch. A similar issue is raised here #93 
- it's unlikely that i'll implement statistics though, it's not really something i had in mind when building this app.

**Reply by mary-loulou** [reply]
Hello @lyqht 
It would be absolutely amazing if your tool supported dynamic QR-codes. 
The codes it creates are just really really really nice. :-)

Would adding simple click statistics make it so much more complicated?
For us, it would be amazing (though not absolutely necessary) to have that kind of information.
